torch/_inductor/kernel/bmm.py

```diff
@@ -43,12 +43,6 @@ def _is_large_block_for_cpu(m, n, k):
     return m * n > 2**12
 
 
-def bmm_configs(m, n, k, *, device_type):
-    if device_type == "cpu":
-        return mm_configs(m, n, k, scale=0.5, exclude=_is_large_block_for_cpu)
```

Comment:
Did this scale=0.5/_is_large_block_for_cpu thing matter?  I don't see it below.

Reply:
We moved it to mm_common as mm_config_kwargs and is imported where used instead of being redefined in multiple files

New usage:
```
if use_triton_template(layout):
    for config in bmm_configs(
        m, n, k, **mm_config_kwargs(ir.get_device_type(mat1))
    ):
```